x-pack/plugins/transform/server/lib/alerting/register_transform_alerting_rules.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Logger } from 'kibana/server';
+import type { AlertingPlugin } from '../../../../alerting/server';
+// import { registerTransformHealthRuleType } from './transform_health_rule_type';
```

Comment:
Is the commented out code in this file needed?

Reply:
Removed in https://github.com/elastic/kibana/pull/112277/commits/6c27a241a1860d001eb6bcd84fec2cb8ec504b0c